evap/evaluation/templates/evap_course_edit_js.html

```diff
@@ -2,48 +2,90 @@
                 }
-            };
+            }
         }
+    }
 
-        function getContributorCount() {
-            return $("#contribution_table select").length;
+    function makeDisabledClickHandler() {
+        return function () {
+            // don't do anything on disabled buttons
+            if ($(this).hasClass("disabled"))
+                return false;
         }
+    }
 
-        function assignClickHandlers() {
-            for(var i = 0; i < getContributorCount(); i++)
-                $("#id_contributions-" + i + "-responsible").click(makeClickHandler(i, getContributorCount()));
+    function uncheckButtons(selector){
+        $(selector).prop("checked", false);
+        $(selector).parent().removeClass("active");
+    }
+    function checkButtons(selector){
+        $(selector).prop("checked", true);
+        $(selector).parent().addClass("active");
+    }
+    function disableButtons(selector){
+        $(selector).parent().addClass("disabled");
+    }
+    function enableButtons(selector){
+        $(selector).parent().removeClass("disabled");
+    }
+
+    function getContributorCount() {
+        return $("#contribution_table select").length;
+    }
+
+    function assignClickHandlers() {
+        for(var i = 0; i < getContributorCount(); i++)
+        {
+            $("input[name=contributions-" + i + "-responsibility][value=RESPONSIBLE]").parent().click(makeClickHandler(i, getContributorCount(), 'RESPONSIBLE'));
+            $("input[name=contributions-" + i + "-responsibility][value=EDITOR]").parent().click(makeClickHandler(i, getContributorCount(), 'EDITOR'));
+            $("input[name=contributions-" + i + "-responsibility][value=CONTRIBUTOR]").parent().click(makeClickHandler(i, getContributorCount(), 'CONTRIBUTOR'));
+            $("input[name=contributions-" + i + "-comment_visibility]").parent().click(makeDisabledClickHandler());
         }
+    }
 
     $(document).ready(function() {
-        assignClickHandlers()
+        assignClickHandlers();
 
         // apply initial settings
         for(var i = 0; i < getContributorCount(); i++){
-            if($("#id_contributions-" + i + "-responsible").is(":checked")) {
-                $("#id_contributions-" + i + "-can_edit").prop("checked", true);
-                $("#id_contributions-" + i + "-can_edit").prop("disabled", true);
+            if($("input[name=contributions-" + i + "-responsibility][value=RESPONSIBLE]").is(":checked")) {
+                // disable comment buttons for responsible contributor
+                disableButtons("input[name=contributions-" + i + "-comment_visibility]");
+                // disable all other responsible buttons
+                for(var j = 0; j < getContributorCount(); j++){
+                    if (j != i)
+                        disableButtons("input[name=contributions-" + j + "-responsibility][value=RESPONSIBLE]");
+                }
```

Comment:
responsible contributors already have "all" saved in the db. should i make a second override here?